docsource/message.js.md

```diff
@@ -0,0 +1,47 @@
+# Message
+
+    var gcm = require("gcm");
+
+A message contains the data that is sent to the Android device.
+It is constructed with an options object, which can contain any of the given keys.
+All of the keys are optional.
+If no object is passed, it is the same as passing no keys.
+
+See the [GCM server reference](https://developer.android.com/google/gcm/server-ref.html#table1) for details on some of the options.
+
+ - `collapseKey` identifies a group of messages that can be collapsed.
+   If you send several notifications with the same notification key to a device that is offline, it will only show the most recent when it comes back online.
+ - When `delayWhileIdle` is set to `true`, it indicates that the message shuold not be sent until the device becomes active.

```

Comment:
"When `delayWhileIdle` is set to `true`, it indicates that the message **shuold** not be sent until the device becomes active."

Should be "should" :smile: